cli/features.go

```diff
@@ -38,10 +36,6 @@ func featuresList() *cobra.Command {
 		Use:     "list",
 		Aliases: []string{"ls"},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			err := cliui.ValidateColumns(featureColumns, columns)
-			if err != nil {
-				return err
-			}
```

Comment:
One downside of this change is that we have to round-trip to the server for a response instead of exiting immediately on bad input.

Would it make sense to expose a `cliui.ValidateTableColumns([]codersdk.Derp{}, columns)` function?

Reply:
I'm hoping to automate this whole process of validating columns and adding the columns/sort flags in a future PR. I just removed this because it didn't match any of the other commands, but your comment is valid. I can add it back if you want or we could wait until we have better output utilities.

Reply:
Sounds good. My opinion is that we’re not in a hurry to restore the early check. It’s a small QoL improvement but it can wait for the refactor just fine.